src/services/event.ts

```diff
@@ -124,6 +126,8 @@ interface FindEventHelperBlobOptions {
   regions: number[] | undefined;
   fallbackValue?: string;
   allowNull?: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  scopes: object;
```

Comment:
import { WhereOptions } from 'sequelize';
...
scopes: WhereOptions[]

Reply:
Added the whereoptions.